Cargo.toml

```diff
@@ -49,6 +49,7 @@ members = [
   "oak_restricted_kernel_interface",
   "oak_restricted_kernel_sdk",
   "oak_sev_guest",
+  "oak_sev_snp_attestation_report",
```

Comment:
I think the naming should be consistent with `oak_sev_guest` (i.e. no `snp`, or add that to both).

Reply:
As far as I've been able to tell, we always refer to the concept as "SEV-SNP" therefore I think those 3 extra letters can add valuable semantic context for a reader to better know where they are. I'd rename to `oak_sev_snp_guest` in a separate PR (b/322303263) to avoid clutter.